src/Domain/Behaviour/BehaviourGateway.php

```diff
@@ -230,4 +230,28 @@ public function queryBehaviourRecordsByPerson(QueryCriteria $criteria, $gibbonSc
 
             return $this->runQuery($query, $criteria);
     }
+
+    public function getBehaviourID($gibbonSchoolYearID, $gibbonBehaviourID)
+    {
+        $data = ['gibbonSchoolYearID' => $gibbonSchoolYearID, 'gibbonBehaviourID' => $gibbonBehaviourID];
+        $sql = 'SELECT gibbonBehaviour.*, student.surname AS surnameStudent, student.preferredName AS preferredNameStudent, creator.surname AS surnameCreator, creator.preferredName AS preferredNameCreator, creator.title FROM gibbonBehaviour JOIN gibbonPerson AS student ON (gibbonBehaviour.gibbonPersonID=student.gibbonPersonID) JOIN gibbonPerson AS creator ON (gibbonBehaviour.gibbonPersonIDCreator=creator.gibbonPersonID) WHERE gibbonSchoolYearID=:gibbonSchoolYearID AND gibbonBehaviourID=:gibbonBehaviourID ORDER BY date DESC';
+        
+        return $this->db()->select($sql, $data);
```

Comment:
Since these are both get methods, it's best to use selectOne, for consistency. It also makes the code in the script easier, as you don't need to call fetch on it.